content/auth/capabilities.textile

```diff
@@ -117,25 +114,25 @@ If a set of capabilities are requested, then the Ably Token will be assigned the
 Using the following example, an API key exists with the listed capabilities. If an "Ably Token":/auth/token#tokens is requested and specifies a set of capabilities, then the resulting token will only receive those capabilities that intersect. The capabilities of a token cannot exceed those of the issuing API key.
 
 ```[javascript]
-// API key capabilities:
-{
-  "chat:*": ["publish", "subscribe", "presence"],
-  "status": ["subscribe", "history"],
-  "alerts": ["subscribe"]
-}
-
-// Token request that specifies capabilities:
-auth.requestToken({ capability: {
-  "chat:bob": ["subscribe"], // only "subscribe" intersects
-  "status": ["*"], // "*"" intersects with "subscribe"
-  "secret": ["publish", "subscribe"] // key does not have access to "secret" channel
-}}, tokenCallback)
-
-// Resulting token capabilities:
-{
-  "chat:bob": ["subscribe"],
-  "status": ["subscribe", "history"]
-}
+  // API key capabilities:
+  {
+    'chat:*': ['publish', 'subscribe', 'presence'],
+    'status': ['subscribe', 'history'],
+    'alerts': ['subscribe']
+  }
+
+  // Token request that specifies capabilities:
+  const requestToken = auth.requestToken({ capability: {
```

Comment:
```suggestion
  const tokenDetails = await auth.requestToken({ capability: {
```

Reply:
https://github.com/ably/docs/pull/2119/commits/536a260694f5faa322f0122cf78af5946118313a

Updated. Thanks for spotting this mistake